app/src/main/java/nerd/tuxmobil/fahrplan/congress/alarms/AlarmReceiver.kt

```diff
@@ -0,0 +1,131 @@
+package nerd.tuxmobil.fahrplan.congress.alarms
+
+import android.app.PendingIntent
+import android.app.PendingIntent.FLAG_ONE_SHOT
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import androidx.annotation.VisibleForTesting
+
+import androidx.core.net.toUri
+
+import info.metadude.android.eventfahrplan.commons.logging.Logging
+
+import nerd.tuxmobil.fahrplan.congress.autoupdate.UpdateService
+import nerd.tuxmobil.fahrplan.congress.contract.BundleKeys
+import nerd.tuxmobil.fahrplan.congress.exceptions.BuilderException
+import nerd.tuxmobil.fahrplan.congress.extensions.withExtras
+import nerd.tuxmobil.fahrplan.congress.notifications.NotificationHelper
+import nerd.tuxmobil.fahrplan.congress.repositories.AppRepository
+import nerd.tuxmobil.fahrplan.congress.schedule.MainActivity.Companion.createLaunchIntent
+import nerd.tuxmobil.fahrplan.congress.utils.PendingIntentCompat.FLAG_IMMUTABLE
+
+class AlarmReceiver : BroadcastReceiver() {
+
+    companion object {
+        const val ALARM_UPDATE = "nerd.tuxmobil.fahrplan.congress.ALARM_UPDATE"
+        @VisibleForTesting
+        private const val ALARM_SESSION = "nerd.tuxmobil.fahrplan.congress.ALARM_SESSION"
```

Comment:
`@VisibleForTesting` hides the property from usage from other runtime classes. It acts like a `private` modifier. Still the test class can access. You cannot put `private` here. Hence, the test failed.

Reply:
Please rebase before your next push.

Reply:
Fixed in new Push :+1: